docs/src/pages/components/stack/InteractiveStack.js

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import Stack from '@material-ui/core/Stack';
+import Grid from '@material-ui/core/Grid';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
We are migrating away from this API. using the sx prop should be enough in most of the cases.